financeager/entries.py

```diff
@@ -70,10 +75,17 @@ def prettify(element, *, default_category):
 
     # Define order of listed properties
     if recurrent:
-        properties = ("name", "value", "frequency", "start", "end", "category")
+        properties: tuple[str, str, str, str, str, str] = (
```

Comment:
just definy properties as a list[str] before the if-else block if that makes things easier